Dockerfile

```diff
@@ -6,9 +6,9 @@ COPY package*.json ./
 
 ARG API_URL
 
-ENV API_URL ${API_URL}
+ENV API_URL = "https://ivpztrmt3p.eu-west-1.awsapprunner.com/swagger#/Test%20API/testConnection"
```

Comment:
nope, don't hardcode this and the API URL doesn't need the swagger path etc

Reply:
ok will change that now